src/js/disability-benefits/686/containers/ConfirmationPage.jsx

```diff
@@ -21,29 +21,58 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
+    /*
     const { submission, data } = this.props.form;
     const { response } = submission;
     const name = data.veteranFullName;
```

Comment:
Added a [comment](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/9827#issuecomment-384287289)